docs/src/content/docs/pt-br/guides/pages.mdx

```diff
@@ -0,0 +1,174 @@
+
+**tipo:** `StarlightPageFrontmatter`
+
+Especifique as [propriedades de frontmatter](/pt-br/reference/frontmatter/) para esta página, similar ao frontmatter em páginas Markdown.
+O [`title`](/pt-br/reference/frontmatter/#title-obrigatório) é requerido e todas as outras propriedades são opcionais.
+
+As seguintes propriedades diferem do frontmatter Markdown:
+
+- O [`slug`](/pt-br/reference/frontmatter/#slug) não é suportado e é automaticamente definido com base na URL da página customizada.
+- A opção [`editUrl`](/pt-br/reference/frontmatter/#editurl) requer uma URL para exibir um link de edição.
+- A propriedade [`sidebar`](/pt-br/reference/frontmatter/#sidebar) para customizar como a página aparece em [grupos de links autogerados](/pt-br/reference/configuration/#sidebar) não está disponível. Páginas usando o componente `<StarlightPage />` não fazem parte de uma coleção e não podem ser adicionadas a um grupo de sidebar autogerado.
+
+##### `sidebar`
+
+**tipo:** `SidebarEntry[]`  
+**ão:** a barra lateral gerada baseado na [configuração global `sidebar`](/pt-br/reference/configuration/#sidebar)
+
+Provê uma barra lateral de navegação customizada para esta página.
+Caso não seja definido, a página usará a barra lateral global padrão.
+
+Por exemplo, a seguinte página sobrescreve o sidebar padrão com um link para a página inicial e um grupo de links para diferentes constelações.
+A página atual na barra lateral é definida usando a propriedade `isCurrent` e um `badge` opcional foi adicionado a um item de link.
+
+```astro {3-13}
+<StarlightPage
+	frontmatter={{ title: 'Orion' }}
+	sidebar={[
+		{ label: 'Início', href: '/' },
+		{
+			label: 'Constelações',
+			items: [
+				{ label: 'Andromeda', href: '/andromeda/' },
+				{ label: 'Orion', href: '/orion/', isCurrent: true },
+				{ label: 'Ursa Minor', href: '/ursa-minor/', badge: 'Stub' },
+			],
+		},
+	]}
+>
+  Conteúdo de exemplo.
+</StarlightPage>
+```
+
+##### `hasSidebar`
+
+**tipo:** `boolean`  
+**padrão:** `false` se [`frontmatter.template`](/pt-br/reference/frontmatter/#template) é `'splash'`, caso contrário é `true`
+
+Controle se a barra lateral deve ser exibida nesta página.
+
+##### `headings`
+
+**tipo:** `{ depth: number; slug: string; text: string }[]`  
+**padrão:** `[]`
+
+Provê um array com todos os títulos na página.
+Starlight irá gerar o sumário da página a partir destes títulos se fornecidos.
+
+##### `dir`
+
+**tipo:** `'ltr' | 'rtl'`  
+**padrão:** a direção de escrita para a localidade atual
+
+Define a direção de escrita para o conteúdo da página.
+
+##### `lang`
+
+**tipo:** `string`  
+**padrão:** a linguagem da localidade atual
+
+Define a tag de linguagem BCP-47 para o conteúdo da página, por exemplo, `en`, `zh-CN`, ou `pt-BR`.
+
+##### `isFallback`
+
+**tipo:** `boolean`  
+**padrão:** `false`
+
+Indica se esta página está usando [conteúdo fallback](/pt-br/guides/i18n/#fallback-content) porque não há tradução para a linguagem atual.
```

Comment:
```suggestion
Indica se esta página está usando [conteúdo fallback](/pt-br/guides/i18n/#conteúdo-de-fallback) porque não há tradução para a linguagem atual.
```

> I did dig a little more and ended with just one last broken link.

Thanks for the updates 🙌 

It looks like the last broken link is this one as in `/pt-br/guides/i18n/#fallback-content`, the `fallback-content` does not exist but is instead [`conteúdo-de-fallback`](https://starlight.astro.build/pt-br/guides/i18n/#conte%C3%BAdo-de-fallback) as it's based on the section title which is translated too.